pydantic_ai_slim/pydantic_ai/_mcp.py

```diff
@@ -121,3 +207,50 @@ def map_from_sampling_content(
         return messages.TextPart(content=content.text)
     else:
         raise NotImplementedError('Image and Audio responses in sampling are not yet supported')
+
+
+def map_from_mcp_resource(mcp_resource: mcp_types.Resource) -> Resource:
```

Comment:
This (and the others) can be private right. Edit: never mind, this is already a private module